README.md

```diff
@@ -1477,7 +1464,7 @@ A useful workaround is to change the above post-hook to:
 ### Reporting bugs and contributing code
 
 - Want to report a bug or request a feature? Let us know in the `#package-ecosystem` channel on [Slack](http://community.getdbt.com/), or open [an issue](https://github.com/dbt-labs/dbt-utils/issues/new)
-- Want to help us build dbt-utils? Check out the [Contributing Guide](https://github.com/dbt-utils/dbt-core/blob/HEAD/CONTRIBUTING.md)
+- Want to help us build dbt-utils? Check out the [Contributing Guide](https://github.com/dbt-labs/dbt-utils/blob/main/CONTRIBUTING.md)
```

Comment:
looks like I pulled in a few main changes, can cleanup but most of these look ok?

Reply:
Yeah this should all be fine!